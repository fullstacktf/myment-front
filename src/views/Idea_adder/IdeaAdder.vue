<template></template>

<script>
export default {
  name: IdeaAdder,
};
</script>

<style>
</style>
