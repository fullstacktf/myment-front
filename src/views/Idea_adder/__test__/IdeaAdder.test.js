import IdeaAdder from '../IdeaAdder.vue';
import { shallowMount, createLocalVue } from '@vue/test-utils';
import Buefy from 'buefy';
import mock from '../../../test/mockBack';

describe('IdeaAdder', () => {
  let localVue;
  beforeEach(() => {
    const Vue = createLocalVue();
    Vue.use(Buefy);
    localVue = Vue;
  });
  test('render component', () => {
    const wrapper = shallowMount(IdeaAdder, {
      localVue,
    });
    expect(wrapper.html()).toMatchSnapshot();
  });

  test('Check props', () => {});

  test('check Contents props', () => {});
});
