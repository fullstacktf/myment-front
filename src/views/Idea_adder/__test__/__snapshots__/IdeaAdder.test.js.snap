// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IdeaAdder render component 1`] = `"<div class=\\"ideaContainer\\"><h1>idea</h1></div>"`;
